src/aks-preview/HISTORY.rst

```diff
@@ -16,6 +16,11 @@ Pending
 * Enable v2 decorator pattern.
 * Fix container name inconsistency for private clusters in kollect command.
```

Comment:
Also move these pending items to the new version section.

Reply:
Done